lib/src/layer/tile_layer/tile_update_transformer.dart

```diff
@@ -75,4 +98,68 @@ abstract class TileUpdateTransformers {
       },
     );
   }
+
+  /// Suppresses tile updates with less inter-event spacing than [duration]
+  ///
+  /// This may improve performance, and reduce the number of tile requests, but
+  /// at the expense of UX: new tiles will not be loaded until [duration] after
+  /// the final tile load event in a series. For example, a fling gesture will
+  /// not load new tiles during its animation, only at the end. Best used in
+  /// combination with the cancellable tile provider, for even more fine-tuned
+  /// optimization.
+  ///
+  /// Implementation follows that in
+  /// ['package:stream_transform'](https://pub.dev/documentation/stream_transform/latest/stream_transform/RateLimit/debounce.html),
+  /// with `leading` and `trailing` set `true`.
+  ///
+  /// Also see [throttle].
+  ///
+  /// ---
+  ///
+  /// {@macro tut-ignore_tap}
+  static TileUpdateTransformer debounce(Duration duration) {
+    const leading = true;
```

Comment:
what's the reason to handle leading events?

Reply:
Good spot, mis-reasoned about that. Resolved.